src/components/common/health-declaration-signed/health-declaration-signed.html

```diff
@@ -15,7 +15,7 @@
                  class="gds-radio-button"
                  [class.ng-invalid]="invalid"
                  (change)="healthDeclarationChanged($event.target.value)"
-                 value="true">
+                 value="Signed">
```

Comment:
could we not have just done [value]="true" which i think should leave it as a boolean value? this would then eliminate the need for the enum and the elvis logic in the TS

Reply:
unfortunatly that didnt seem to work :(